### PR TITLE
`curate-maintain` Add content for the Content section

### PR DESCRIPTION
**Pull request by molecula2788**

### CODE REVIEW

File: chapters/curate-maintain/content/reading/README.md [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
-Owner: Adrian Șendroiu
-Reviewer: Ștefan Jumărea
+The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.
 
-the central part is content management, around a repository
+It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.
 
-typically Git & GitHub
+The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.
+It is also recommended to use a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:8]:
```suggestion
It is also recommended using a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.
```

File: chapters/curate-maintain/content/reading/README.md [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
 
-versioning, maintaining up-to-date
+The content should be published under a license.
+An example of a widely used license for public education resources is `Creative Commons`.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:11]: I would rephrase this to state that we strongly encourage (and use) CC licencing

File: chapters/curate-maintain/content/reading/README.md [hunk 3]
```diff
@@ -1,25 +1,35 @@
...
 
-encouraging contributions, integrating changes
+#### Contributions
 
-creating issues, monitoring issues and pull requests
+The content maintainer should foster a welcoming environment for outside contributions.
+This can be achieved by:
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:16]:
```suggestion
This can be achieved by:

```

File: chapters/curate-maintain/content/reading/README.md [hunk 4]
```diff
@@ -1,25 +1,35 @@
...
+- establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:17]:
```suggestion
- Establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)
```

File: chapters/curate-maintain/content/reading/README.md [hunk 5]
```diff
@@ -1,25 +1,35 @@
...
+- labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:18]:
```suggestion
- Labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute
```

File: chapters/curate-maintain/content/reading/README.md [hunk 6]
```diff
@@ -1,25 +1,35 @@
...
+- acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:19]:
```suggestion
- Acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)
```

File: chapters/curate-maintain/content/reading/README.md [hunk 7]
```diff
@@ -1,25 +1,35 @@
...
 
-licensing, ownership
+There should also be an established process for handling contributions whenever they arrive.
+Some steps that the maintainer should take are:
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:22]:
```suggestion
Some steps that the maintainer should take are:

```

File: chapters/curate-maintain/content/reading/README.md [hunk 8]
```diff
@@ -1,25 +1,35 @@
...
+- review the contribution as soon as possible.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:23]:
```suggestion
- Review the contribution as soon as possible.
```

File: chapters/curate-maintain/content/reading/README.md [hunk 9]
```diff
@@ -1,25 +1,35 @@
...
+Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
+Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:25]:
```suggestion
Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.
```

File: chapters/curate-maintain/content/reading/README.md [hunk 10]
```diff
@@ -1,25 +1,35 @@
...
+- engage in a constructive discussion with the contributor to clarify any questions or concerns.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:26]:
```suggestion
- Engage in a constructive discussion with the contributor to clarify any questions or concerns.
```

File: chapters/curate-maintain/content/reading/README.md [hunk 11]
```diff
@@ -1,25 +1,35 @@
...
+Provide feedback and guidance for improvement if the contribution needs changes.
+- ensure that the contribution follows the style guidelines.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:28]:
```suggestion
- Ensure that the contribution follows the guidelines.
```
There can be many guidelines, not necessarily style-related.

File: chapters/curate-maintain/content/reading/README.md [hunk 12]
```diff
@@ -1,25 +1,35 @@
...
+This can be partially automated using linters and automated checkers, together with features like `GitHub Actions`.
+- if applicable, test the contribution to ensure it works as intended.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:30]:
```suggestion
- If applicable, test the contribution to ensure it works as intended.
```

File: chapters/curate-maintain/content/reading/README.md [hunk 13]
```diff
@@ -1,25 +1,35 @@
...
+- thank the contributor for their effort.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:31]:
```suggestion
- Thank the contributor for their effort.
```

File: chapters/curate-maintain/content/reading/README.md [hunk 14]
```diff
@@ -1,25 +1,35 @@
...
-the process of creating a repository from zero
-how the structure should be created;
-initial contributions;
-submission via PRs and review process
+#### Creating a repository
 
-the process of improving a repository via contributions;
-how the contribution + review process should happen;
-steps to be done, reviewers, assignees
+If a new content repository needs to be created from scratch, a good starting point is to clone the [oer-template](https://github.com/open-education-hub/oer-template) repository.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:35]: I would add a short phrase on what the `oer-template` contains. Just say that it has a basic repository structure with all the tools included, and any kind of content can be added on top of that.

Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:35]:
```suggestion
If a new content repository needs to be created from scratch, a good starting point is to clone the [`oer-template`](https://github.com/open-education-hub/oer-template) repository.
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
Looks good @molecula2788, see the comments.

File: chapters/curate-maintain/content/reading/README.md [hunk 15]
```diff
@@ -1,25 +1,38 @@
 ### Content
 
-Owner: Adrian Șendroiu
-Reviewer: Ștefan Jumărea
+The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.
 
-the central part is content management, around a repository
+It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:5]: What exactly does `such a resource` refer to?

File: chapters/curate-maintain/content/reading/README.md [hunk 16]
```diff
@@ -1,25 +1,38 @@
...
 
-typically Git & GitHub
+The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.
+It is also recommended using a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.
 
-versioning, maintaining up-to-date
+The content should be published under a license.
+At `Open Education Hub` we employ the `Creative Commons` license ourselves and strongly recommend its usage.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:11]:
```suggestion
At `Open Education Hub` we employ the [`Creative Commons`](https://en.wikipedia.org/wiki/Creative_Commons_license) license ourselves and strongly recommend its usage.
```

File: chapters/curate-maintain/content/reading/README.md [hunk 17]
```diff
@@ -1,25 +1,38 @@
...
+#### Contributions
 
-creating issues, monitoring issues and pull requests
+The content maintainer should foster a welcoming environment for outside contributions.
+This can be achieved by:
 
-licensing, ownership
+- Establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file)
+- Labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute
+- Acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place)
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:20]:
```suggestion
- Establishing clear contribution guidelines (for example in a `CONTRIBUTING.md` file).
- Labeling issues with tags such as `help wanted` or `good first issue`, to help newcomers find tasks to contribute.
- Acknowledging the work of contributors (for example in the repository `README.md` file or in any other suitable place).
```
Add dots at the end of sentences.

File: chapters/curate-maintain/content/reading/README.md [hunk 18]
```diff
@@ -1,25 +1,38 @@
...
-steps to be done, reviewers, assignees
+- Review the contribution as soon as possible.
+Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
+Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.
+- Engage in a constructive discussion with the contributor to clarify any questions or concerns.
+Provide feedback and guidance for improvement if the contribution needs changes.
+- Ensure that the contribution follows the guidelines.
+This can be partially automated using linters and automated checkers, together with features like `GitHub Actions`.
+- If applicable, test the contribution to ensure it works as intended.
+- Thank the contributor for their effort.
```
Comment (StefanJum) [inline chapters/curate-maintain/content/reading/README.md:33]: Maybe add that the maintainer should make sure contributions are merged, and do not end up lingering for too long?

**Review by StefanJum — CHANGES_REQUESTED** [review]
Looks good @molecula2788, just few minor comments.

File: chapters/curate-maintain/content/reading/README.md [hunk 19]
```diff
@@ -1,25 +1,38 @@
...
 
-Owner: Adrian Șendroiu
-Reviewer: Ștefan Jumărea
+The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.
 
-the central part is content management, around a repository
+It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.
 
-typically Git & GitHub
+The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.
```
Comment (RazvanN7) [inline chapters/curate-maintain/content/reading/README.md:7]:
```suggestion
The content is typically organized in a versioned system control repository such as `Git`, which has a series of advantages like version control, branching, history, etc.
```

**Review by RazvanN7 — APPROVED** [review]
Awesome work! Take care of the remaining bits and this should be good to go.

File: chapters/curate-maintain/content/reading/README.md (hunk 15 above)
Comment (molecula2788) [reply chapters/curate-maintain/content/reading/README.md:5]: An open education resource.

File: chapters/curate-maintain/content/reading/README.md [hunk 20]
```diff
@@ -1,25 +1,38 @@
...
-submission via PRs and review process
+There should also be an established process for handling contributions whenever they arrive.
+Some steps that the maintainer should take are:
 
-the process of improving a repository via contributions;
-how the contribution + review process should happen;
-steps to be done, reviewers, assignees
+- Review the contribution as soon as possible.
+Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
+Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.
```
Comment (Alex-deVis) [inline chapters/curate-maintain/content/reading/README.md:27]:
```suggestion
- Review the contribution as soon as possible.

  Depending on the team size, this can be done by the maintainer, or it could be delegated to other members of the content development team.
  Tools like `GitHub` have features that can help with this: an issue can have a person designated as assignee, while a pull request can have one or multiple reviewers and assignees.
```

File: chapters/curate-maintain/content/reading/README.md (hunk 18 above)
Comment (Alex-deVis) [inline chapters/curate-maintain/content/reading/README.md:33]: Buy him something nice once in a while.

File: chapters/curate-maintain/content/reading/README.md (hunk 15 above)
Comment (StefanJum) [reply chapters/curate-maintain/content/reading/README.md:5]: Got it, it's fine.

**Review by StefanJum — APPROVED** [review]
All good now, thank you @molecula2788

**Comment by RazvanN7** [comment]
@molecula2788 please squash commits.

**Comment by Alex-deVis** [comment]
Yes, @molecula2788, please squash your commits.

**Review by Alex-deVis — APPROVED** [review]
